lib/Basics/Utf8Helper.cpp

```diff
@@ -218,7 +223,7 @@ bool Utf8Helper::setCollatorLanguage(std::string_view lang,
   return true;
 }
 
-std::string Utf8Helper::getCollatorLanguage() {
```

Comment:
I do remember we've agreed upon using `getName()` always. At creation time locale has to be adjusted.